BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Responses/RegisterResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BackendApartmentReservation.DataContracts.DataTransferObjects.Responses
+{
+    public class RegisterResponse
+    {
+        [Required(AllowEmptyStrings = false)]
+        public int Id { get; set; }
```

Comment:
Required wont do anything there, especially with integers :D default value, if ID is not provided, will be 0. To make it trully valid, you could do something like:
```
[Required]
public int? Id {get; set;}
```

But that may be a bit of an overkill:D

Reply:
Also, since its a response DTO, I doubt we need validation at all

Reply:
And one more thing:DD Do you think we need to return all the data about the customer at registration? Maybe only ID is enough, and then frontend will query on demand using this ID?

Reply:
The response returns only Id, am I wrong?

Reply:
Sorry, my bad here. I guess in my eyes two github files merged:D